### PR TITLE
Documentation for Core APIs

### LINKED ISSUE #16: Documentation setup for core-js
## Description:
Adding documentations for Core-JS APIs and usage

## Things to do:

- [x] Docstrings for core APIs
- [x] Generate documentation

### PR DESCRIPTION
**Pull request by yathishram**
## Description:
This PR adds documentation for APIs and generates documentation.

## Added:
List the added features
* Doc Strings for APIs
* API useage information

## Modified:

List the modified features
* Updates Ceramic URL.
* Updates ceramic definition URLs.

## Todo:
List the TODO features

- [x] Generate documentation
- [x] Add doc strings
- [ ] Update function names and structuring if needed.

### CODE REVIEW

File: src/SafientCore.ts [hunk 1]
```diff
@@ -100,9 +120,11 @@ export class SafientCore {
     }
   };
 
+ 
   /**
-   * API 3:getLoginUser
-   *
+   * This API is used to get the login information of the user
+   * @param did - DID of the user.
+   * @returns - Returns User information
```
Comment (koshikraj) [inline src/SafientCore.ts]: remove `Returns`

File: src/SafientCore.ts [hunk 2]
```diff
@@ -510,8 +581,13 @@ export class SafientCore {
           throw new Error('Error while getting onChain data')
         }
       }
-
-      //Onchain function
+      
+      /**
+       * This API gets claim data for the safe from onChain.
+       * @ignore
```
Comment (koshikraj) [inline src/SafientCore.ts]: Why do we need `ignore` here?

File: src/SafientCore.ts (hunk 1 above)
Comment (yathishram) [reply src/SafientCore.ts]: Done

File: src/SafientCore.ts (hunk 2 above)
Comment (yathishram) [reply src/SafientCore.ts]: Oh, That was by mistake will remove that.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**